Keas.Mvc/Startup.cs

```diff
@@ -218,11 +218,7 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory, IHostApplicationLifetime appLifetime)
         {
-            app.UseAllElasticApm(Configuration);
-            
-            // setup logging
-            app.ConfigureStackifyLogging(Configuration);

```

Comment:
Did you mean to delete these?

Reply:
Agreed I would think you'd still need the serilog entry at least.

Reply:
`ConfigureStackifyLogging()` is called in `Program.cs`
`loggerFactory.AddSerilog()` is the old way of doing it, and is handled internally by `IHostBuilder.UseSerilog()` in `Program.cs`